make_test_images/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "make_test_images"
+version = "0.1.0"
```

Comment:
If this is to live within the same repo as SDK in the long term, then I'd like to have the version number match. This would require an update to .github/workflows/ci.yml.